src/validation/helpers/writeInfoInline.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Writes information about if a type is required and if it can be empty inline
+ */
+export default function writeInfoInline(type, canBeEmpty, required) {
+    const empty = canBeEmpty ? '?' : '';
+    return type && `${required ? `<${empty}${type}>` : `[${empty}${type}]`}`;
```

Comment:
The idea is that `type` will only ever be the empty string, and because of that we will return the empty string. Could be discussed if it would be better to not have such implicit assumptions.